javascript/javascript1/week3-Homework

sumankaundinya opened the pull request.

jason-vasilev reviewed (changes requested):
Hi, Suman. Did you by chance work directly on `main`? I am not able to comment on your code as it seems like the only changes are those few `<!-- minor change to enable pull request -->` disclaimers.
To solve this, I'd recommend you to close the PR, delete the files from `main`, make a new branch and add them there - just then try to push that new branch and open the PR. It's not critical - if you'd prefer, we can do it together tomorrow before the session.

sumankaundinya commented:
> Hi, Suman. Did you by chance work directly on `main`? I am not able to comment on your code as it seems like the only changes are those few `<!-- minor change to enable pull request -->` disclaimers. To solve this, I'd recommend you to close the PR, delete the files from `main`, make a new branch and add them there - just then try to push that new branch and open the PR. It's not critical - if you'd prefer, we can do it together tomorrow before the session.

sumankaundinya commented:
Updates on PR Issue:

Hi Jason,

Based on your feedback, I followed these steps to correct my PR:

Understanding the Issue

I realized that my original PR didn’t show meaningful changes.
As per your suggestion, I needed to delete the files from main and create a new branch for my work.
Branch Cleanup

I attempted to delete the unnecessary branches (javascript-javascript1-week3 and javascript/javascript1/week3) from both GitHub and my local machine.
I was able to delete these branches from GitHub but couldn’t delete main since it's the default branch.
Locally, I still saw the deleted branches.
Creating a Proper Branch

I renamed javascript/javascript1/week3 to javascript/javascript1/week3-Homework.
I staged and committed my files in this new branch.
I successfully pushed the branch to GitHub.
PR Creation Attempt

I tried creating a PR from javascript/javascript1/week3-Homework to main.
GitHub showed "There isn’t anything to compare", meaning it saw no differences.
I then deleted the files from main to make the differences more visible.
I also switched the base branch back and forth to refresh the comparison.
Current Issue

After switching the base, the PR still doesn’t show any files.
However, when I manually check the branch javascript/javascript1/week3-Homework on GitHub, all my files are there.
Looking forward to your guidance!

Thanks,
Suman

jason-vasilev commented:
Oh no! I think I might have confused you even more. Only the files for the current week's exercises had to be removed, and not all other things from `main`. Lets sit at the session and see how can we roll back the changes and fix this.
